bundles/org.openhab.binding.enocean/src/main/resources/ESH-INF/thing/channels.xml

```diff
@@ -449,4 +449,86 @@
 		<category>thermostat</category>
 	</channel-type>
 
+	<channel-type id="smokeDetection">
+		<item-type>Switch</item-type>
+		<label>Smoke Detected</label>
+		<description>Smoke detection sensor state.</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="sensorFault">
+		<item-type>Switch</item-type>
+		<label>Sensor Fault Mode</label>
+		<description>Sensor activated fault mode.</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="maintenanceStatus">
+		<item-type>Switch</item-type>
+		<label>Maintenance neccessary</label>
+		<description>Sensor maintenance status.</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="saHumidityRange">
+		<item-type>Switch</item-type>
+		<label>Out of Humidity Range</label>
+		<description>Sensor analysis status: humidity out of range</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="saTemperatureRange">
+		<item-type>Switch</item-type>
+		<label>Out of Temperature Range</label>
+		<description>Sensor analysis status: temperature out of range</description>
+		<state readOnly="true" />
+	</channel-type>
+
+	<channel-type id="timeSinceLastMaintenance">
+		<item-type>Number</item-type>
+		<label>Weeks Since Last Maintenance</label>
```

Comment:
Consider to keep the labels short. 

Would this make more sense to have it as a quantity unit so that people can choose to have it in days, months or whatever

Reply:
Good point, changed to QuantityType and shortend the labels as far as I could. Open for any other suggestions.